x-pack/plugins/observability/public/components/shared/exploratory_view/views/series_views.tsx

```diff
@@ -20,10 +19,8 @@ export function SeriesViews({
 }) {
   return (
     <div ref={seriesBuilderRef}>
-      <EuiPanel color="subdued">
-        <SeriesEditor />
-        <AddSeriesButton />
-      </EuiPanel>
+      <SeriesEditor />
+      <AddSeriesButton />
     </div>
   );
```

Comment:
might as well just delete this component, i don't see it having any purpose now.